### PR TITLE
[Bugfix] #7344 Add emojies to comments

### PR DESCRIPTION
**Pull request by kryzanivska-nastya**
[#7344](https://github.com/ita-social-projects/GreenCity/issues/7344),[#7776](https://github.com/ita-social-projects/GreenCity/issues/7776),[#7775](https://github.com/ita-social-projects/GreenCity/issues/7775)
**result** 

https://github.com/user-attachments/assets/1d01bbb3-66e1-4f00-9f79-c8d86f73e5f4

### CODE REVIEW

File: src/app/chat/component/chat/chat.component.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Message } from '../../model/Message.model';
 import { FormControl } from '@angular/forms';
 import { SocketService } from '../../service/socket/socket.service';
 import { UserService } from '@global-service/user/user.service';
+import { EmojiService } from '@global-service/emoji/emoji.service';
 
 @Component({
```
Comment (Lokankara) [inline src/app/chat/component/chat/chat.component.ts:11]: `ChatComponent` is not declared in any Angular module
Unresolved variable emptyChatBack in template: 26:
``

File: src/app/chat/component/chat/chat.component.ts [hunk 2]
```diff
@@ -74,8 +76,8 @@ export class ChatComponent implements OnInit, AfterViewChecked {
     this.showEmojiPicker = !this.showEmojiPicker;
   }
 
-  addEmoji(event) {
-    const newValue = this.messageControl.value ? this.messageControl.value + event.emoji.native : event.emoji.native;
+  addEmoji(event): void {
```
Comment (Lokankara) [inline src/app/chat/component/chat/chat.component.ts]: Parameter 'event' implicitly has an 'any' type, but a better type may be inferred from usage.

File: src/app/main/component/comments/components/comment-textarea/comment-textarea.component.ts [hunk 3]
```diff
@@ -37,6 +38,7 @@ export class CommentTextareaComponent implements OnInit, AfterViewInit, OnChange
   isImageUploaderOpen = false;
   showImageControls = false;
   isFirstImageLoaded = false;
```
Comment (Lokankara) [inline src/app/main/component/comments/components/comment-textarea/comment-textarea.component.ts]: Unused field isFirstImageLoaded
Unused method onDropdownBlur

File: src/app/main/service/emoji/emoji.service.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EmojiService {
```
Comment (Lokankara) [inline src/app/main/service/emoji/emoji.service.ts]: The `EmojiService` can be refactored into a functional utility to simplify its implementation and remove the DI mechanism if it is only performing a straightforward operation:
`export const insertEmoji = (currentText: string, emoji: string): string =>  `${currentText}${emoji}`;`

File: src/app/main/component/comments/components/comment-textarea/comment-textarea.component.ts [hunk 5]
```diff
@@ -142,7 +145,24 @@ export class CommentTextareaComponent implements OnInit, AfterViewInit, OnChange
     }, 0);
   }
```
Comment (Lokankara) [inline src/app/main/component/comments/components/comment-textarea/comment-textarea.component.ts:162]: The `ngAfterViewInit` can be refactored into smaller, well-defined methods For better readability and maintainability

**Review by Lokankara — APPROVED** [review]

**Review by Tadei-Mykola — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**